manifests/t/TELUS/TELUSBusinessConnectPhone/23.1.1.45971/TELUS.TELUSBusinessConnectPhone.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: TELUS.TELUSBusinessConnectPhone

```

```suggestion
PackageIdentifier: TELUS.BusinessConnectPhone
```
Best practice is to not repeat the publisher. After making this change, the folder and file names will need to be updated also